src/Microsoft.IdentityModel.Protocols/Configuration/HttpDocumentRetriever.cs

```diff
@@ -64,8 +78,15 @@ public HttpDocumentRetriever(HttpClient httpClient)
             }
             catch (Exception ex)
             {
-                LogHelper.Throw(string.Format(CultureInfo.InvariantCulture, LogMessages.IDX10804, address), typeof(IOException), EventLevel.Error, ex);
-                return null;
+                if (File.Exists(address))
```

Comment:
Good point. I got rid of GenericDocumentRetriver since it was doing the same thing as HttpDocumentRetriever except reading from the file. How about just DocumentRetriever or GenericDocumentRetriever?